gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs/api-runtime-logs.component.html

```diff
@@ -43,10 +43,11 @@
     ></api-runtime-logs-quick-filters>
   }
 
-  @if (apiLogsSubject$ | async; as response) {
+  @if (isMessageApi() != null && apiLogsSubject$ | async; as logs) {
```

Comment:
```suggestion
  @if (apiLogsSubject$ | async; as logs) {
```
From what I understand,`isMessageApi()` will always be either be true or false. Is it necessary in the `@if`?

Reply:
done